controllers/reviews.controller.js

```diff
@@ -0,0 +1,22 @@
+const { selectReviews, selectReviewById } = require("../models/reviews.model");
+
+exports.getReviews = (req, res) => {
+  selectReviews()
+    .then((reviews) => {
+      res.status(200).send(reviews);
+    })
+    .catch((err) => {
+      console.log(err);
```

Comment:
be sure to pass your err along to the error handling middleware I'm shortly going to ask you to write tests and solution for :)